src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestBindCallGen.generated.txt

```diff
@@ -0,0 +1,122 @@
+// <auto-generated/>
+#nullable enable
+
+using System.Linq;
```

Comment:
What is using `System.Linq`? In general, it would be better if we didn't use it unless we really had to. We've been removing a lot of usages of it for perf and size on disk reasons.

Reply:
That would be checks to see if we should return `default` in the `Get` method.. https://github.com/dotnet/runtime/blob/f4ade8200bb6efd8f0d73d0cd24100e2ca45ad34/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/TestGetCallGen.generated.txt#L18. I could generate  a helper method instead.

Reply:
Resolved in https://github.com/dotnet/runtime/pull/83614.